### ISSUE TITLE
[Suggestion] Forbid new users to raise issues

### ISSUE THREAD

**Issue by FA810** [Codewars/codewars.com#861]
It's happening to me quite often to see some 7kyu _noobs_ with a total of 4 completed katas (and often quitting CW after a month) raising issues over katas just because they can't solve it, perhaps because they feel the urge of changing the input parameters when it's unnecessary or whatnot. Not happy about this, they even down-vote you enraged if you dare to close their issue...

Considering that I've read somewhere issues might hinder the beta-kata process approval, why don't we forbid this and limit it to experienced users only?
Is it possible to get issue-closing without comment or to have something like "A kata editor closed this issue"?

**Comment by DonaldKellett** [comment]
Sounds good to me, I second that :smile:  Maybe issue-raising should be remade into a User Privilege that is unlocked at 500 Honor or so.

**Comment by 10XL** [comment]
You will never satisfy everyone and you'll get stuck in a rabbit hole trying to do so. Usually if someone downvotes your kata or even only writes comments in their code putting your kata at fault, you can respond to it in a comment and if it makes sense, people should agree with you. See https://github.com/Codewars/codewars.com/issues/535

**Comment by jhoffner** [comment]
I don't know that these would be privileges, since IMO its kind of messed up to say that your allowed to say something is good but not bad, and to limit the number of people who can say things are good or bad is just going to cause the beta process to be slower. 

I think ideally the system would need to be smarter on the backend to know when to selectively ignore or discount the effect a new user's vote has on the beta process. 

However how often does this really come up per each challenge? If you anger 1 or even 2 new users who give you a down vote, that shouldn't be preventing any kata from getting out of beta if it is otherwise good content. It seems that there is enough "padding" built-in to the minimums to allow for this type of thing without having to over-complicate things too much. 

If this isn't the case, please provide an example kata that this phenomenon is happening to.

**Comment by FA810** [comment]
The one I remember the most is about a kata in Shell where I added "\r\n" to the output to make it work.
You (Codewars Platform Developers) have changed this console behaviour, so now the expected output is *without* the trailing "\r\n" and the tests in Shell have suddenly become wrong. 

Surely the "\r\n" was not visible.
I've tried to explain this, got downvoted...

OFF-Topic: while we are at it, can you review my Java translation to one of your katas? It's a 5 kyu:
https://www.codewars.com/kata/convert-string-to-camel-case/discuss

**Comment by jhoffner** [comment]
> Surely the "\r\n" was not visible. I've tried to explain this, got downvoted...

Honestly some people are just stubborn and no amount of business logic is going to code ourselves around that. Fortunately that one downvote isn't going to amount to anything. 

> OFF-Topic: while we are at it...

funny enough I just deployed a fix that allows me to approve this without having to fork your translation. Normally you have to add a language example for each language that gets translated (if the description contains examples) - or else the example won't show at all. It is now changed so that it defaults to the first language found. 

In other words when someone is taking this challenge in java,  they will be shown the javascript examples within the description since they come first and no java examples are defined.
